catalog/task/skopeo-inspect/0.1/skopeo-inspect.yaml

```diff
@@ -0,0 +1,30 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: skopeo-inspect
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: release
+spec:
+  description: >-
+    Tekton task that executes `skopeo inspect` on an image and saves the result
+  params:
+    - name: imageURL
+      description: Image URL to inspect
+      type: string
+  workspaces:
+    - name: output
+      description: The workspace to place the skopeo-inspect.json file in
+  steps:
+    - name: skopeo-inspect
+      image: quay.io/skopeo/stable:v1.7.0
+      script: |
+        if [[ -z "$(params.imageURL)" ]]
+        then
+          exit 1
+        fi
+
+        skopeo inspect "$(params.imageURL)" > $(workspaces.output.path)/skopeo-inspect.json
```

Comment:
Should we use results for this? If we are not going to rely on external gitops repositories for the strategy configuration, we won't need workspaces in general.

Reply:
We had a similar idea when creating the [sanity-inspect-image](https://github.com/redhat-appstudio/build-definitions/blob/main/task/sanity-inspect-image/0.1/sanity-inspect-image.yaml) task, but gave up on it because the inspection output could be too large for the Tekton results size limit.
Also, I'd recommend using the`--no-tags` option for the inspect command to avoid performance issues (see the explanation in the  [build definitions PR 35](https://github.com/redhat-appstudio/build-definitions/pull/35)).